NitroxLauncher/Properties/Settings.Designer.cs

```diff
@@ -1,18 +1,18 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
```

Comment:
Can something be done to prevent system language making unncessary changes like this?

Reply:
As far as I know you need the English language pack installed.